chaos-tests/src/test/java/ChaosIT.java

```diff
@@ -0,0 +1,161 @@
+
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import io.fabric8.junit.jupiter.api.KubernetesTest;
+import io.fabric8.junit.jupiter.api.LoadKubernetesManifests;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.Version;
+import io.fabric8.kubernetes.client.dsl.NamespaceableResource;
+import io.fabric8.kubernetes.client.dsl.PodResource;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@KubernetesTest
+@LoadKubernetesManifests(value = { "checker-infra.yaml", "control-infra.yaml" })
+public class ChaosIT {
+  protected Logger logger = LoggerFactory.getLogger(getClass().getName());
+
+  KubernetesClient client;
+
+  String checkerImage = Optional.ofNullable(System.getenv("CHECKER_IMAGE")).orElse("chaos-test-checker:latest");
+  String controlImage = Optional.ofNullable(System.getenv("CONTROL_IMAGE")).orElse("chaos-test-control:latest");
+  String chaosTest = Optional.ofNullable(System.getenv("CHAOS_TEST")).orElse("network-delay.yaml");
+
+  @BeforeEach
+  void beforeEach() throws Exception {
+    logger.info("BeforeEach execution");
+    // this needs to be executed before the pods are started
+    client.configMaps().inNamespace(client.getNamespace()).withName("chaos-test").delete();
+  }
+
+  @AfterEach
+  void afterEach() throws Exception {
+    logger.info("AfterEach execution");
+
+    try {
+      String checkerLogs = checkerSelector().getLog();
+      System.out.println("*** Checker Logs ***\n" + checkerLogs);
+      System.out.println("******");
+    } catch (Exception e) {
+      // ignore
+    }
+
+    try {
+      String controlLogs = controlSelector().getLog();
+      System.out.println("*** Control Logs ***\n" + controlLogs);
+      System.out.println("******");
```

Comment:
In some places I see Slf4j logger getting used, in some places `System.out` . Is it possible to use only one way?